Cargo.toml

```diff
@@ -403,7 +403,7 @@ color-eyre = { git = "https://github.com/eyre-rs/eyre.git", rev = "e5d92c3" }
 deltalake = { git = "https://github.com/delta-io/delta-rs", rev = "57795da" }
 ethnum = { git = "https://github.com/ariesdevil/ethnum-rs", rev = "4cb05f1" }
 openai_api_rust = { git = "https://github.com/datafuse-extras/openai-api", rev = "819a0ed" }
-orc-rust = { git = "https://github.com/youngsofun/datafusion-orc", rev = "1745375" }
+orc-rust = { git = "https://github.com/youngsofun/datafusion-orc", rev = "fd028c2" }
```

Comment:
Can we fork this repo to datafuse-extras?

Reply:
There is already a repo there: https://github.com/datafuse-extras/datafusion-orc